app/src/main/java/com/sunlightlabs/congress/services/LegislatorService.java

```diff
@@ -186,6 +187,16 @@ else if (!json.isNull("id"))
                 legislator.office = role.getString("office");
             if (!role.isNull("phone"))
                 legislator.phone = role.getString("phone");
+
+            // committee memberships stored on roles, in separate fields
+            // the list of committees per-member will be sorted by committee ID
+            // so returning them as one big flat list will work
+            legislator.committees = new ArrayList<Committee>();
+            if (!role.isNull("committees"))
+                legislator.committees.addAll(CommitteeService.committeesFromArray(role.getJSONArray("committees"), false));
+            if (!role.isNull("subcommittees"))
+                legislator.committees.addAll(CommitteeService.committeesFromArray(role.getJSONArray("subcommittees"), true));
```

Comment:
Parsing of committee membership for a particular member is controlled by `LegislatorService`, but to consolidate committee-specific parsing code (and because it's running the same code twice), this calls back out to a `CommitteeService` method.